haskell-indexer-pipeline-ghckythe-wrapper/src/Language/Haskell/Indexer/Args.hs

```diff
@@ -61,10 +65,11 @@ data Flags = Flags
     , flagPrependPathPrefix   :: !(Maybe Text)
     , flagKeepTempPathPrefix  :: !Bool
     , flagOverridePgmP        :: !(Maybe FilePath)
+    , flagOutput              :: !(Maybe FilePath)
```

Comment:
Needs a comment if this is a single path or a directory. My hunch based on shallow review is that now this is a dir, with files created based on module name + id. If so, this behavior should be commented, ideally in the flag description.

But I can imagine some users might want to direct output to a file/pipe/etc, so maybe having two alternatives (outfile vs outdir) would be nice.

For example, usecases where the plugin'd GHC is invoked as a subprocess as part of a larger service, I can imagine the output would be a single pipe, which the service could stream out without keeping temporary files around (entries can be quite huge). I imagine latter to be a viable scenario when Google-internal infra is updated to use the source plugin. Btw I left Google so @ivan444 to judge such internal things.

Reply:
It used to be a single file but I was having issues with pipes so I changed it. If someone wants to refactor it later to be suitable to use with a single pipe then it would not be difficult to do so.

Reply:
The ability to specify a directory is probably important to make the plugin easy to use with build systems that build multiple modules concurrently.